### ISSUE TITLE
I'm changing the UI but it is not showing any effect, So in which file I need to change??

### ISSUE THREAD

**Issue by aagamshah0304** [SilverDecisions/SilverDecisions#198]

**Comment by mwasiluk** [comment]
It depends on what are you trying to change.
Please take a look at https://github.com/SilverDecisions/SilverDecisions/wiki/Developer's-guide
Hope that helps!

**Comment by aagamshah0304** [comment]
> It depends on what are you trying to change.
> Please take a look at https://github.com/SilverDecisions/SilverDecisions/wiki/Developer's-guide
> Hope that helps!

I had seen this but no progress and I am changing the toolbar UI but it is not affecting even after I m commenting the InitToolbar() function in app.js file and changing in toolbar.html file also!! 
I am doing since 4 days but no results in UI!!
And thanks!! I'll see it again!

**Comment by mwasiluk** [comment]
Have you built your project using gulp after making changes?

**Comment by aagamshah0304** [comment]
Yes but I'm using gulp serve because using gulp it also search for test cases and the test cases are failed using gulp, so I m using gulp serve!!

**Comment by mwasiluk** [comment]
Which branch are you trying to build? Dev branch should not fail.
https://travis-ci.org/github/SilverDecisions/SilverDecisions

**Comment by aagamshah0304** [comment]
> Which branch are you trying to build? Dev branch should not fail.
> https://travis-ci.org/github/SilverDecisions/SilverDecisions

I'm using dev branch only 
And I want to change the shape of the terminal node into parellel lines but the path is generated automatically how can i change?? Can you tell??
